descriptive-link-5effbb.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 5effbb
+name: Link text is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+- CSS Styling
+
```

Comment:
Should "Language" be an input aspect here?
Still up for discussion in https://github.com/act-rules/act-rules.github.io/issues/671

Reply:
As you said, when we reach a conclusion in #671 we'll probably have to go through all the rules and add Language to those that need it. Agree that this will be one of those.